dex/src/main/scala/com/wavesplatform/dex/api/http/entities/HttpOrderBook.scala

```diff
@@ -43,7 +46,10 @@ object HttpOrderBook {
 
   }
 
-  private def formatValue(value: BigDecimal, decimals: Int): String = new java.text.DecimalFormat(s"0.${"0" * decimals}").format(value)
+  private def formatValue(value: BigDecimal, decimals: Int): String = {
+    val symbols = new DecimalFormatSymbols(Locale.US)
```

Comment:
There is a lot of code inside the constructor. Let's cache the value somewhere.

Reply:
resolved